h/templates/accounts/deleted.html.jinja2

```diff
@@ -0,0 +1,6 @@
+{% extends "h:templates/accounts/base.html.jinja2" %}
+
+{% block page_title %}{% trans %}Account deleted{% endtrans %}{% endblock %}
+
+{% set form_message %}{% trans %}Your account has been deleted.{% endtrans %}{% endset %}
+{% set signup_message = "Don't have a Hypothesis account?" %}
```

Comment:
One slightly unfortunate aspect of the experience is that if you click the back button here, eg. if you're attempting to go back to the home page, you get a "page not found" message since of course the account deletion route is not available to the now logged-out user.

Reply:
Hmm. Could we do something to the browser's history to fix this? Maybe with JavaScript?

Reply:
You can't clear the history for the current tab from JS. I think the right solution is probably to improve the 4xx page to be more useful. Right now it doesn't differentiate between page not found and lack of authorization for example. I know this was intentional originally, but it isn't user friendly.